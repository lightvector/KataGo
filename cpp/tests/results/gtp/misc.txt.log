: Running with following config:
allowResignation = true
cudaUseFP16 = false
cudaUseNHWC = false
forDeterministicTesting = true
lagBuffer = 1.0
logAllGTPCommunication = true
logFile = tests/results/gtp/misc.txt.log
logSearchInfo = true
logSearchInfoForChosenMove = false
logTimeStamp = false
logToStderr = false
maxPlayouts = 10000
maxTimePondering = 60.0
maxVisits = 100
nnRandSeed = forTesting
nnRandomize = false
numSearchThreads = 1
openclUseFP16 = false
ponderingEnabled = false
resignConsecTurns = 3
resignThreshold = -0.90
rootSymmetryPruning = false
rules = tromp-taylor
searchFactorAfterOnePass = 0.50
searchFactorAfterTwoPass = 0.25
searchFactorWhenWinning = 0.40
searchFactorWhenWinningThreshold = 0.95
searchRandSeed = forTesting
trtUseFP16 = false

: GTP Engine starting...
: KataGo v1.16.4
: Using TrompTaylor rules initially, unless GTP/GUI overrides this
: Using 1 CPU thread(s) for search
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 19 * 19 exactly
: Cuda backend thread 0: Found GPU NVIDIA RTX A5000 memory 25425608704 compute capability major 8 minor 6
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 19 nnYLen 19
: Initializing board with boardXSize 19 boardYSize 19
: Loaded config configs/gtp_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: logDir = 
: Config override: logFile = tests/results/gtp/misc.txt.log
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: nnRandSeed = forTesting
: Config override: nnRandomize = false
: Config override: numSearchThreads = 1
: Config override: openclUseFP16 = false
: Config override: rootSymmetryPruning = false
: Config override: searchRandSeed = forTesting
: Config override: trtUseFP16 = false
: Model name: g170-b6c96-s175395328-d26788732
: GTP ready, beginning main protocol loop
: Controller: kata-list-params
: = analysisWideRootNoise analysisIgnorePreRootHistory genmoveAntiMirror antiMirror humanSLProfile allowResignation ponderingEnabled delayMoveScale delayMoveMax chosenMovePrune chosenMoveSubtract chosenMoveTemperature chosenMoveTemperatureEarly chosenMoveTemperatureHalflife chosenMoveTemperatureOnlyBelowProb conservativePass cpuctExploration cpuctExplorationBase cpuctExplorationLog cpuctUtilityStdevPrior cpuctUtilityStdevPriorWeight cpuctUtilityStdevScale drawEquivalentWinsForWhite dynamicScoreCenterScale dynamicScoreCenterZeroWeight dynamicScoreUtilityFactor enableMorePassingHacks enablePassingHacks endgameTurnTimeDecay fillDameBeforePass fpuLossProp fpuParentWeight fpuParentWeightByVisitedPolicy fpuParentWeightByVisitedPolicyPow fpuReductionMax futileVisitsThreshold graphSearchCatchUpLeakProb graphSearchRepBound humanSLChosenMoveIgnorePass humanSLChosenMovePiklLambda humanSLChosenMoveProp humanSLCpuctExploration humanSLCpuctPermanent humanSLOppExploreProbWeightful humanSLOppExploreProbWeightless humanSLPlaExploreProbWeightful humanSLPlaExploreProbWeightless humanSLRootExploreProbWeightful humanSLRootExploreProbWeightless ignoreAllHistory ignorePreRootHistory lagBuffer lcbStdevs maxPlayouts maxPlayoutsPondering maxTime maxTimePondering maxVisits maxVisitsPondering midgameTimeFactor midgameTurnPeakTime minPlayoutsPerThread minVisitPropForLCB nnPolicyTemperature noResultUtilityForWhite noisePruneUtilityScale noisePruningCap numSearchThreads numVirtualLossesPerThread obviousMovesPolicyEntropyTolerance obviousMovesPolicySurpriseTolerance obviousMovesTimeFactor overallocateTimeFactor playoutDoublingAdvantage playoutDoublingAdvantagePla policyOptimism rootDesiredPerChildVisitsCoeff rootDirichletNoiseTotalConcentration rootDirichletNoiseWeight rootEndingBonusPoints rootFpuLossProp rootFpuReductionMax rootNoiseEnabled rootNumSymmetriesToSample rootPolicyOptimism rootPolicyTemperature rootPolicyTemperatureEarly rootPruneUselessMoves rootSymmetryPruning searchFactorAfterOnePass searchFactorAfterTwoPass staticScoreUtilityFactor subtreeValueBiasFactor subtreeValueBiasFreeProp subtreeValueBiasTableNumShards subtreeValueBiasWeightExponent treeReuseCarryOverTimeFactor uncertaintyCoeff uncertaintyExponent uncertaintyMaxWeight useGraphSearch useLcbForSelection useNoisePruning useNonBuggyLcb useUncertainty valueWeightExponent wideRootNoise winLossUtilityFactor
: Controller: komi 50.5
: = 
: Controller: get_komi
: = 50.5
: Controller: komi -23
: = 
: Controller: get_komi
: = -23
: Controller: version
: = 1.16.4+g170-b6c96-s175M
: Controller: kata-get-models
: = [{"internalName":"g170-b6c96-s175395328-d26788732","maxBatchSize":8,"name":"tests/models/g170-b6c96-s175395328-d26788732.bin.gz","usesHumanSLProfile":false,"usingFP16":"false","version":8}]
: Controller: kata-set-param humanSLProfile rank_10k
: ? Could not set params: WARNING: humanSLProfile is specified as config param but model(s) don't use it: tests/models/g170-b6c96-s175395328-d26788732.bin.gz

: Controller: kata-get-param allowResignation
: = true
: Controller: kata-get-param ponderingEnabled
: = false
: Controller: kata-get-param delayMoveMax
: = 1e+06
: Controller: kata-get-param maxVisits
: = 100
: Controller: kata-set-param delayMoveMax 1234
: = 
: Controller: kata-set-param allowResignation false
: = 
: Controller: kata-get-param allowResignation
: = false
: Controller: kata-get-param ponderingEnabled
: = false
: Controller: kata-get-param delayMoveMax
: = 1234
: Controller: kata-get-param maxVisits
: = 100
: Controller: kata-set-param delayMoveMax 12345
: = 
: Controller: kata-set-param maxVisits 4321
: = 
: Controller: kata-get-param allowResignation
: = false
: Controller: kata-get-param ponderingEnabled
: = false
: Controller: kata-get-param delayMoveMax
: = 12345
: Controller: kata-get-param maxVisits
: = 4321
: GPU -1 finishing, processed 0 rows 0 batches
: All cleaned up, quitting
