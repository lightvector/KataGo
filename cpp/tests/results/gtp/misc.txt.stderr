KataGo v1.16.4
Using TrompTaylor rules initially, unless GTP/GUI overrides this
Initializing board with boardXSize 19 boardYSize 19
Loaded config configs/gtp_example.cfg and/or command-line and query overrides
Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
Model name: g170-b6c96-s175395328-d26788732
GTP ready, beginning main protocol loop
